homeassistant/components/tradfri/base_class.py

```diff
@@ -43,18 +44,18 @@ class TradfriBaseClass(Entity):
     _attr_should_poll = False
 
     def __init__(
-        self, device: Command, api: Callable[[str], Any], gateway_id: str
+        self, device: Device, api: Callable[[str], Any], gateway_id: str
```

Comment:
The api accepts a Command or a list of commands.

Reply:
Corrected.